test/e2e-go/features/catchup/catchpointCatchup_test.go

```diff
@@ -297,14 +297,17 @@ outer:
 	}
 	log.Infof("done catching up!\n")
 
-	timer = time.NewTimer(10 * time.Second)
+	timer = time.NewTimer(100 * time.Second)
 	for {
 		status, err = secondNodeRestClient.Status()
 		a.NoError(err)
 		primStatus, err := primaryNodeRestClient.Status()
 		a.NoError(err)
-		log.Infof("currend round: %d, primary round: %d, current-sync time and time since last round %d, %d, prim-sync time and time since last round: %d, %d",
-			status.LastRound, primStatus.LastRound, status.CatchupTime, status.TimeSinceLastRound, primStatus.CatchupTime, primStatus.TimeSinceLastRound)
+		log.Infof("currend round: %d, primary round: %d, current-sync time and time since last round %f, %f, prim-sync time and time since last round: %f, %f, 2nd node catchpoint: %v",
+			status.LastRound, primStatus.LastRound,
+			time.Duration(status.CatchupTime).Seconds(), time.Duration(status.TimeSinceLastRound).Seconds(),
+			time.Duration(primStatus.CatchupTime).Seconds(), time.Duration(primStatus.TimeSinceLastRound).Seconds(),
+			*status.Catchpoint)
```

Comment:
THIS IS REALLY CONFUSING TO ME: on my machine, the second node has a non zero sync time, even when the round is matching with primary node... for example:

```
    testingLogger.go:38: time="2023-03-09T14:19:20.300401 -0500" level=info msg="currend round: 58, primary round: 58, current-sync time and time since last round 97.832242, 3.049511, prim-sync time and time since last round: 0.000000, 3.364625, 2nd node catchpoint: " file=catchpointCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchpointCatchup line=306
    testingLogger.go:38: time="2023-03-09T14:19:20.552098 -0500" level=info msg="currend round: 59, primary round: 59, current-sync time and time since last round 98.083984, 0.176102, prim-sync time and time since last round: 0.000000, 0.201082, 2nd node catchpoint: " file=catchpointCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchpointCatchup line=306
    testingLogger.go:38: time="2023-03-09T14:19:22.569221 -0500" level=info msg="currend round: 59, primary round: 59, current-sync time and time since last round 100.101189, 2.193302, prim-sync time and time since last round: 0.000000, 2.218260, 2nd node catchpoint: " file=catchpointCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchpointCatchup line=306
    testingLogger.go:38: time="2023-03-09T14:19:22.821213 -0500" level=info msg="currend round: 59, primary round: 59, current-sync time and time since last round 100.353119, 2.445231, prim-sync time and time since last round: 0.000000, 2.470257, 2nd node catchpoint: " file=catchpointCatchup_test.go function=github.com/algorand/go-algorand/test/e2e-go/features/catchup.TestBasicCatchpointCatchup line=306
```